UI/GameSettingsScreen.cpp

```diff
@@ -1072,6 +1082,25 @@ UI::EventReturn GameSettingsScreen::OnChangeMacAddress(UI::EventParams &e) {
 	return UI::EVENT_DONE;
 }
 
+UI::EventReturn GameSettingsScreen::OnChangeRemoteISOSubdir(UI::EventParams &e) {
+	//Conform to HTTP standards
+	for (size_t i = 0; i < g_Config.sRemoteISOSubdir.length(); i++)
+	{
+		if (g_Config.sRemoteISOSubdir[i] == ' ') g_Config.sRemoteISOSubdir[i] = '+';
+		if (g_Config.sRemoteISOSubdir[i] == '\\') g_Config.sRemoteISOSubdir[i] = '/';
+		if (g_Config.sRemoteISOSubdir[i] == '?') {
+			g_Config.sRemoteISOSubdir.erase(i); //truncate for safety
+			break;
+		}
+	}
+	//Make sure it begins and ends with /
+	if (g_Config.sRemoteISOSubdir[0] != '/')
+		g_Config.sRemoteISOSubdir = "/" + g_Config.sRemoteISOSubdir;
+	if (g_Config.sRemoteISOSubdir[g_Config.sRemoteISOSubdir.length() - 1] != '/')
```

Comment:
This seems unnecessarily limiting.  Maybe someone will publish a PHP script you can host on a local server which generates something PPSSPP reads, but uses a query string, like a frontend.

-[Unknown]